test/server/configuration_impl_test.cc

```diff
@@ -614,6 +614,7 @@ TEST(InitialImplTest, LayeredRuntime) {
   NiceMock<MockOptions> options;
   NiceMock<Server::MockInstance> server;
   InitialImpl config(bootstrap, options, server);
+  config.initAccesslog(bootstrap, server);
```

Comment:
Is this needed everywhere where InitialImpl is used? Looks like most tests have an empty admin section config...